public/js/server.js

```diff
@@ -326,6 +326,42 @@ Server.deleteArchive = function (arcId, callback) {
         .catch((error) => LRR.showErrorToast("Error while deleting archive", error));
 };
 
+/**
+ * Sends a POST request to queue a find_duplicates job,
+ * detecting archive duplicates based on their thumbnail hashes.
+ */
+Server.findDuplicates = function () {
+    Server.callAPI(`/api/find_dupes`, "POST",
+        "Queued up a job to find duplicates! Stay tuned for updates or check the Minion console.",
+        "Error while sending job to Minion:",
+        (data) => {
+            // Disable the buttons to avoid accidental double-clicks.
+            $(".find-duplicates").prop("disabled", true);
```

Comment:
I think it'd also be good to feature a spinner of some form in the UI to indicate the job is in progress.  

Now of course that all kinda goes away if the user reloads the page while the job is running -- I wonder if we shouldn't store the minion job ID somewhere so it can be detected on a reload. (Either on the controller side with the database, or just with localstorage? Controller is probably better but that'd be a bit annoying with the generic API call)